_example/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+
+	cf "github.com/essentialkaos/go-confluence/v5"
+)
+
+func main() {
```

Comment:
I'm not sure that it will be really helpful for users if we provide just one example with couple of methods.

Reply:
I plan to move these basic usage examples into `example_test.go`. In this case, these examples will be shown on [pkg.go.dev](https://pkg.go.dev/github.com/essentialkaos/go-confluence/v5). So, I can merge it like that and then refactor them as a part of preparing for release.

Reply:
@andyone Sorry I will remove the example that is just for testing

Reply:
[5fd41b4](https://github.com/essentialkaos/go-confluence/pull/52/commits/5fd41b488acd05aa840270b43395f7d38f72a929) Done